ru-ru/tmux-ru.html.markdown

```diff
@@ -0,0 +1,253 @@
+---
+category: tool
+tool: tmux
+contributors:
+  - ["mdln", "https://github.com/mdln"]
+translators:
+  - ["Davydov Anton", "https://github.com/davydovanton"]
+filename: LearnTmux.txt
```

Comment:
Interesting. It should be `leartmux-ru.md`, but for some reason it is also `txt` in the English version. @adambard?